orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/model/StageExecutionImpl.java

```diff
@@ -343,6 +343,23 @@ public void setLastModified(@Nullable LastModifiedDetails lastModified) {
     this.lastModified = lastModified;
   }
 
+  /**
+   * Additional tags to be used with stage metrics. This is useful to add extra dimensions to the
+   * metrics recorded for built-in or custom stages.
+   */
+  private Map<String, String> metricTags;
```

Comment:
a bit of a nit, but perhaps
```suggestion
  private Map<String, String> additionalMetricTags;
```
to match the description?

Reply:
done